applications/openshift/scc/scc_limit_net_raw_capability/rule.yml

```diff
@@ -27,7 +27,26 @@ ocil_clause: 'NET_RAW does not exist or is configured in defaultAddCapabilities
 ocil: |-
     Inspect each SCC returned from running the following command:
     <pre>$ oc get scc</pre>
-    Review each SCC and determine that <tt>NET_RAW</tt> is either
+    Review each SCC and determine that either <tt>NET_RAW</tt> or <tt>ALL</tt> is either
     completely disabled as a list entry under <tt>requiredDropCapabilities</tt>,
-    or that <tt>NET_RAW</tt> is only enabled to a small set of
+    or that either <tt>NET_RAW</tt> or <tt>ALL</tt> is only enabled to a small set of
     containers and SCCs.
+
+{{%- if product == "ocp4" %}}
+warnings:
+    - general: |-
+        {{{ openshift_cluster_setting("/apis/security.openshift.io/v1/securitycontextconstraints") | indent(8) }}}
+{{%- endif %}}
+
+template:
+    name: yamlfile_value
+    vars:
+        ocp_data: "true"
+        filepath: /apis/security.openshift.io/v1/securitycontextconstraints
+        yamlpath: ".items[:]['requiredDropCapabilities']"
```

Comment:
Hi @evgenyz can you help us out here? We have a list of SCCs and then the `requiredDropCapabilities` is a list, too. The `requiredDropCapabilities` attributes is a list itself, too, it might look like:
```
  requiredDropCapabilities:
  - KILL
  - MKNOD
  - SETUID
  - SETGID
```
So I tried to adjust the YAML-path like this:
```
".items[:]['requiredDropCapabilities'][:]"
```
but this causes an error in the scanner:
```
I: oscap:     Opening file '/kubernetes-api-resources/apis/security.openshift.io/v1/securitycontextconstraints'.
I: oscap:     YAML path '.items[:]['requiredDropCapabilities'][:]' points to a multi-dimensional structure (sequence containing another sequence)
I: oscap:     Test 'oval:ssg-test_scc_drop_container_capabilities:tst:1' requires that at least one object defined by 'oval:ssg-object_scc_drop_container_capabilities:obj:1' exists on the system.
I: oscap:     0 objects defined by 'oval:ssg-
```
Can you suggest if we can do anything?

Reply:
As stupid as it might sound, this would help: `.items[:]['requiredDropCapabilities','#'][:]`. And even `.items[:]['requiredDropCapabilities','#']` might be enough, as ending `[:]` is usually redundant.

Reply:
Result would be: 
```xml
           <ind-sys:value datatype="record">
              <field name="required^drop^capabilities">KILL</field>
              <field name="required^drop^capabilities">MKNOD</field>
            </ind-sys:value>
            <ind-sys:value datatype="record">
              <field name="required^drop^capabilities">SETGUID</field>
            </ind-sys:value>
            ...
```

Reply:
This '#' is kind of a bug. Or an edge case. But right now, if you want an array behid just one key you have to pretend that you're interested in multiple keys (i.e. keys selection path section type). Of course this would work for multiple keys as well, '#' is just a dummy key.

Reply:
To be precise, if you want an array, behind single key, which is also in array and you're collecting all of them ([:]['key','dummy-key'][:]). Array index, for example, would work without this hack ([0]['key'][:]).

Reply:
Also, `type='string'` is redundant, as default type is always string.